### PR TITLE
[DF-3277] - added makefile targets for pypi release

### PR DESCRIPTION
**Pull request by jherlitz-r7**
- adding makefile targets for pypi release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -57,4 +57,18 @@ deploy: tag
 		docker push komand/python-$${F}-plugin:$(VERSION); \
 		docker push komand/python-$${F}-plugin:$(MINOR_VERSION); \
 		docker push komand/python-$${F}-plugin:$(MAJOR_VERSION); \
-	done
\ No newline at end of file
+	done
+
+# Release targets for PyPi
+packagedeps:
+        python3 -m pip install --user --upgrade setuptools wheel twine
```
Comment (mrinehart-r7) [inline Makefile]: This file should use tabs, instead of spaces - this will cause Make to fail

File: Makefile [hunk 2]
```diff
@@ -57,4 +57,18 @@ deploy: tag
 		docker push komand/python-$${F}-plugin:$(VERSION); \
 		docker push komand/python-$${F}-plugin:$(MINOR_VERSION); \
 		docker push komand/python-$${F}-plugin:$(MAJOR_VERSION); \
-	done
\ No newline at end of file
+	done
+
+# Release targets for PyPi
+packagedeps:
```
Comment (mrinehart-r7) [inline Makefile]: All of these Makefile targets should be using tabs, not spaces

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
-# Komand Python SDK [![Build Status](https://travis-ci.org/rapid7/komand-plugin-sdk-python.svg?branch=master)](https://travis-ci.org/rapid7/komand-plugin-sdk-python)
```
Comment (mrinehart-r7) [inline README.md]: Need to keep this line

File: Makefile (hunk 1 above)
Comment (jschipp-r7) [reply Makefile]: @jherlitz-r7 Always validate your Makefiles with `make -n`

**Pull request merged**